File: jour_12/jour12.c
#include <stdio.h>
int main() {
    int number, result_number;
    do {
        printf("Vous voulez la table de multiplication de quel nombre ? : ");
        result_number = scanf("%d", &number);

        if (result_number == 0) {
            printf("Saisie incorrecte ! Veuillez entrer un nombre entier\n");
            while (getchar() != '\n'); 
        }
    } while (result_number == 0); 

    printf("\n========== Table de multiplication de %d ==========\n", number);
    
    for (int i = 1; i <= 10; i++) {
        printf("%d x %d = %d\n", number, i, number * i);
    }
    return 0;
}

File: jour_12/jour12.py
while True :
    try :
        number = int(input("Vous voulez la table de multiplication de quel nombre ? : "))
        break 
    except ValueError :
        print("saisie incorrect")
print(f"========== table de multiplcaion de {number} ==========")
for i in range(1,11):
    print(f"{number} x {i} = {number*i}\n")